src/test/java/io/anserini/document/WikipediaArticleTest.java

```diff
@@ -36,13 +36,13 @@
 import io.anserini.collection.WikipediaCollection;
 
 
-public class WikipediaArticleTest extends DocumentTest {
+public class WikipediaArticleTest extends DocumentTest<WikipediaArticle> {
   protected static Path tmpPath;
 
   @Before
   public void setUp() throws Exception {
     super.setUp();
-    dType = new TrecwebDocument();
+    dType = new WikipediaArticle("Sample Title", "Sample Content");
```

Comment:
I think we can remove this line.
But I am not sure, if we have to keep it then keep it.

Reply:
Let's keep it for now...